.ci/Jenkinsfile

```diff
@@ -90,8 +90,8 @@ pipeline {
                 // parallelTasks["Windows-Node.js-${version}"] = generateStepForWindows(version: version)
               }
 
-              // Only 12 for the time being
-              parallelTasks["Windows-Node.js-12"] = generateStepForWindows(version: '12')
+              // Only 14 for the time being
+              parallelTasks["Windows-Node.js-14"] = generateStepForWindows(version: '14')
 
```

Comment:
This is a workaround/fix for https://github.com/elastic/apm-agent-nodejs/issues/2061